scripts/create-docs.py

```diff
@@ -0,0 +1,42 @@
+import os
+import re
+import sys
+
+lib_dir = "lib"
+output_resource = sys.argv[1] if len(sys.argv) > 1 else "all-keywords.robot"
+
+# Collect all .robot files
+robot_files = ["keywords.robot"]
+for root, dirs, files in os.walk(lib_dir):
+    for file in files:
+        if file.endswith(".robot"):
+            robot_files.append(os.path.join(root, file))
+
+# Regex pattern to detect keyword definitions
+keyword_pattern = re.compile(
+    r"\n(?:(?!Library|Resource|Variables|Documentation)[A-Z][a-z0-9_]+[\ ]+.*)"
+)
+
+delimiter = "."  # Between filename and the keyword
+
+# Create the combined resource file with prefixed keywords
+with open(output_resource, "w", encoding="utf8") as res_file:
+
+    for robot_file in robot_files:
+        file_prefix = os.path.splitext(os.path.basename(robot_file))[0] + delimiter
+        index_offset = 1  # Initial value = length of \n
```

Comment:
```suggestion
        index_offset = 0
```

if you use previous suggestion (use `^` instead of `\n`)

Reply:
Fixed in 2774df89044aa7b0999e4f32309836c74e45a346